lib/saml_idp/xml_security.rb

```diff
@@ -193,38 +195,54 @@ def validate_doc_embedded_signature(base64_cert, soft = true)
 
         # check digests
         REXML::XPath.each(@sig_element, '//ds:Reference', sig_namespace_hash) do |ref|
-          uri                           = ref.attributes.get_attribute('URI').value
+          uri = ref.attributes.get_attribute('URI').value
+
+          hashed_element = document.at_xpath("//*[@ID='#{uri[1..-1]}']")
+          canon_algorithm  = canon_algorithm REXML::XPath.first(
+            ref,
+            '//ds:CanonicalizationMethod',
+            sig_namespace_hash
+          )
 
-          hashed_element                = document.at_xpath("//*[@ID='#{uri[1..-1]}']")
-          canon_algorithm               = canon_algorithm REXML::XPath.first(ref,
-                                                                             '//ds:CanonicalizationMethod', sig_namespace_hash)
-          canon_hashed_element          = hashed_element.canonicalize(canon_algorithm,
-                                                                      inclusive_namespaces)
+          canon_hashed_element = hashed_element.canonicalize(
+            canon_algorithm,
+            inclusive_namespaces
+          )
 
-          digest_algorithm              = algorithm(REXML::XPath.first(ref, '//ds:DigestMethod'))
+          digest_algorithm = algorithm(REXML::XPath.first(ref, '//ds:DigestMethod'))
 
-          hash                          = digest_algorithm.digest(canon_hashed_element)
-          digest_value                  = Base64.decode64(REXML::XPath.first(ref,
-                                                                             '//ds:DigestValue', sig_namespace_hash).text)
+          hash = digest_algorithm.digest(canon_hashed_element)
+          digest_value = Base64.decode64(REXML::XPath.first(
+            ref,
+            '//ds:DigestValue',
+            sig_namespace_hash
+          ).text)
 
           unless digests_match?(hash, digest_value)
             return soft ? false : (raise ValidationError.new('Digest mismatch'))
           end
         end
 
-        base64_signature        = REXML::XPath.first(@sig_element, '//ds:SignatureValue',
-                                                     sig_namespace_hash).text
-        signature               = Base64.decode64(base64_signature)
-        sig_alg                 = REXML::XPath.first(signed_info_element, '//ds:SignatureMethod',
```

Comment:
I think we could rubocop this

```
Layout/SpaceAroundOperators:
  AllowForAlignment: false
```

Reply:
added in https://github.com/18F/saml_idp/pull/103/commits/8948ddf27c0900a00da08101a59e67c1bb6a9838